### PR TITLE
Bootstrap theme

### PR DESCRIPTION
**Pull request by iamlawrenceonoja**
# What does this PR do? 
This PR adds a new feature to Screenly's playground which is a Bootstrap theme for Screenly’s playground apps. The HTML file can be edited to serve multiple apps in the playground.

# What is the relevant GitHub Issue?

# Checklist

- [x] The code passes linting (for the relevant language)
- [x] I've tested the design in all [supported resolutions](https://github.com/Screenly/playground/blob/master/docs/resolutions.md)
- [x] I've attached relevant screenshots (to speed up code review)
- [x] The final implementation is pixel perfect to the Figma design

# Screenshots
The screenshot below is the web page in landscape resolution using the default image and an example caption.

image

The screenshot below is the web page in portrait resolution using the default image and an example caption.
image

### CODE REVIEW

**Comment by vpetersson** [comment]
There's a structural issue here. `bootstrap/index.html` shouldn't include any CSS. The whole idea is that it should be a re-usable CSS file. So move all CSS out of there and into a dedicated CSS file that can be included.

**Comment by vpetersson** [comment]
When you ready for a formal review, please set this as ready and @iamplugged will do a review.

File: bootstrap/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Inter" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Rubik" rel="stylesheet">
+
+        <link href="main.css" rel="stylesheet">
+    
+    </head>
+    
+    <body>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js"></script>
```
Comment (iamplugged) [inline bootstrap/index.html:21]: Move this script to head

File: bootstrap/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+        <!--Create divs to contain the images and captions-->
+    <div class="container-fluid">
+        
+        <div class="row d-flex justify-content-center mt-2 pb-3 header">
+            <div class="col-4">
+                <img class="playground-logo" width="40%" height="auto" src="images/playground_logo.svg" alt="Screenly logo" style="float:left;">
```
Comment (iamplugged) [inline bootstrap/index.html:28]: Move inline styles to main.css file unless there'a use case

File: bootstrap/index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            </div>
+            <div class="col-4 pt-3 header-text">
+                <span>Instagram Feed</span>
+            </div>
+        </div>
+
+        <div class="row d-flex justify-content-center mt-5 pt-3">   
+            <div id="frame0" class="col-sm-4 frames">
+                <img id="image-zero" class="img-responsive instagram-post" src="images/default_card.svg" alt="Instagram Image">
+                <div class="col-12 caption" id="caption-zero"><b style="color:#0D6EFD;">@screenly</b> Text Placeholder</div>
```
Comment (iamplugged) [inline bootstrap/index.html:38]: Same here. Move inline styles to main.css unless necessary

File: bootstrap/index.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            </div>
+
+            <div id="frame1" class="col-sm-4 frames">
+                <img id="image-one" class="img-responsive instagram-post" src="images/default_card.svg" alt="Instagram Image">
+                <div class="col-12 caption" id="caption-one"><b style="color:#0D6EFD;">@screenly</b> Text Placeholder</div>
```
Comment (iamplugged) [inline bootstrap/index.html:43]: Same. Inline styles

File: bootstrap/main.css [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+  font-family: Inter, sans-serif;
+  position: relative;
+}
+
+.header {
+  color: #fff;
+  font-size: 2.4vh;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  border-bottom: 0.1px groove white;
```
Comment (iamplugged) [inline bootstrap/main.css:16]: px values as decimals are ignored

File: bootstrap/index.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        
+        <title>Instagram Images</title>
+        
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Inter" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Rubik" rel="stylesheet">
```
Comment (iamplugged) [inline bootstrap/index.html:14]: Are you using Rubik anywhere?

File: bootstrap/main.css [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+  height: 100%;
+  overflow: hidden;
+  background: #0d6efd;
+  font-family: Inter, sans-serif;
+  position: relative;
+}
+
+.header {
+  color: #fff;
+  font-size: 2.4vh;
```
Comment (iamplugged) [inline bootstrap/main.css:13]: For font-size, I would recommend using rem

File: bootstrap/index.html (hunk 1 above)
Comment (iamlawrenceonoja) [reply bootstrap/index.html:21]: > Move this script to head

Html lint does not allow script tags in the head tag that’s why it’s there.

File: bootstrap/main.css (hunk 7 above)
Comment (iamlawrenceonoja) [reply bootstrap/main.css:17]: Adnan has not worked on the exact typography rules for this bootstrap theme. This is just a make-shift solution. So I think it's better to leave the unit as it is for now. Then change it when Adnan produces exactly what he has in mind.

Comment (iamlawrenceonoja) [reply bootstrap/main.css:17]: > For font-size, I would recommend using rem

Adnan has not worked on the exact typography rules for this bootstrap theme. This is just a make-shift solution. So I think it's better to leave the unit as it is for now. Then change it when Adnan produces exactly what he has in mind.

**Pull request merged**